backend/src/test/java/com/github/mysterix5/vover/cloud_storage/CloudServiceTest.java

```diff
@@ -17,26 +17,24 @@
 class CloudServiceTest {
 
     @Test
-    void loadMultipleAudioFromCloudAndMerge() {
+    void loadMultipleMp3FromCloudAndMerge() {
         CloudRepository cloudRepository = Mockito.mock(CloudRepository.class);
         CloudService cloudService = new CloudService(cloudRepository);
 
-        var obFile = new File("src/test/resources/cloud_storage/ob.mp3");
-        var dasFile = new File("src/test/resources/cloud_storage/das.mp3");
-        var obDasFile = new File("src/test/resources/cloud_storage/obDas.mp3");
+        var einsFile = new File("src/test/resources/cloud_storage/eins.mp3");
+        var zweiFile = new File("src/test/resources/cloud_storage/zwei.mp3");
+        var einszweiFile = new File("src/test/resources/cloud_storage/einszwei.mp3");
 
-        try (var obStream = new FileInputStream(obFile);
-             var dasStream = new FileInputStream(dasFile);
-             var obDasStream = new FileInputStream(obDasFile)
+        try (var einsStream = new FileInputStream(einsFile);
+             var zweiStream = new FileInputStream(zweiFile);
+             var einszweiStream = new FileInputStream(einszweiFile)
         ) {
-            Mockito.when(cloudRepository.find("ob.mp3")).thenReturn(obStream.readAllBytes());
-            Mockito.when(cloudRepository.find("das.mp3")).thenReturn(dasStream.readAllBytes());
+            Mockito.when(cloudRepository.find("eins.mp3")).thenReturn(einsStream.readAllBytes());
+            Mockito.when(cloudRepository.find("zwei.mp3")).thenReturn(zweiStream.readAllBytes());
 
-            var obDasBytes = obDasStream.readAllBytes();
+            byte[] mergedAudio = cloudService.loadMultipleMp3FromCloudAndMerge(List.of("eins.mp3", "zwei.mp3"));
 
-            AudioInputStream mergedAudio = cloudService.loadMultipleAudioFromCloudAndMerge(List.of("ob.mp3", "das.mp3"));
-
-            assertThat(mergedAudio.readAllBytes()).isEqualTo(obDasBytes);
+            assertThat(mergedAudio.length).isEqualTo(einszweiStream.readAllBytes().length);
         } catch (IOException e) {
             throw new RuntimeException();
         }
```

Comment:
Now with this try catch, wouldn't your test still be green if the Exception is thrown ?